### DISCUSSION TITLE
KAM map of highly-dense dislocation microstructure

### DISCUSSION THREAD

**Discussion by NovakEvana** [mtex-toolbox/mtex#1711] [Ask Anything] [answered]
Hello,

My AM microstructure is known for its high density of GND after a rapid cooling process. Those regions with a high number of dislocations will return with poor indexing or unindexed results during an EBSD measurement (see black regions in the below image). I'm using a KAM map here showing local misorientation that can represent how dense of dislocations is. So how to let MTEX know those regions are highly misoriented (KAM value = 5 degrees) instead of returning with results of 0? Thanks

![image](https://user-images.githubusercontent.com/117882416/211886816-74fd66c0-88e8-4508-81fb-3372bec0dd26.png)

**Comment by kilir** [comment, marked as answer]
Hi,
while I don't think this should be done, if I understand you correctly, you assume that each non-indexed point is actually a piece of crystalline sample (and not a hole or something like that) and should have a high KAM with a value somehow defined? In that case you could set that property for non-indexed points and plot it, OR  simply assign all the point which are not indexed this very values
 e.g. `kam(isnan(kam)) = 4000`. Hope this helps.

Cheers,
Rüdiger

**Reply by NovakEvana** [reply]
Hi Rudiger,

`while I don't think this should be done, if I understand you correctly, you assume that each non-indexed point is actually a piece of crystalline sample (and not a hole or something like that) and should have a high KAM with a value somehow defined?`

Yes, you're right. Considering there was no issue with my material prep, experiment setup, etc. as well as no cracks or defects on my sample, high dislocation density at these regions is the only reason I would think of. Your solution works (I would change to 5.000 instead of 4000 if wanted to draw a distribution chart). Besides that, could you please explain why it shouldn't be done? And could you suggest any other ways (except for TEM) to represent/confirm the high dislocation density present in these regions, especially in my context, where results return with unindexed values?

Thanks Rudiger!

**Reply by NovakEvana** [reply]
Hi @kilir, Could you please have a look at my question? Thanks Rudiger.

**Reply by zmichels** [reply]
@NovakEvana This is something I would not do... simply because it is data fabrication -- there can be other reasons that a pixel is not indexed other than just high dislocation density. If you do it and decide to publish it, I think you would want to be very clear about which parts of the dataset you fabricated and which you didn't. Alternatively, you could leave the data as they are and explain to a reader that you believe those not-indexed points have high values of dislocation density. Just my two cents.

**Reply by NovakEvana** [reply]
Thanks for your inputs, Zach. It's hard to convince readers that those are dislocation regions without direct observation (like TEM). I'm looking for an alternative indicator that can help me to do that.

**Reply by zmichels** [reply]
@NovakEvana I completely understand. Perhaps a "band contrast" or "image quality" map (which is recorded at every point regardless of whether indexed or not) could be useful (?). Presumably those regions of high dislocation densities will have low bc or iq values. May not be useful, but you can check quickly with your data.

**Reply by kilir** [reply]
Hi,
I couldn't agree more with Zach - and that is the reason I mentioned this should not be done. If it is so widely known that the dislocation density is too high to be indexed, everyone should be fine with non-indexed regions. Respectively, if dislocation density is so high that your material loses crystallinity - the part one tries to measure with EBSD -  I would say KAM or GND does even make not much of a sense.

How to measure it? TEM I guess. How to display it qualitatively? If your surfaces have a standard, reproducible quality, like Zach said, bc or a similar measure.

Cheers,
Rüdiger

**Reply by NovakEvana** [reply]
Thanks Zach and Rudiger.
